### PR TITLE
ci: skip broken cri-o tests

### PR DESCRIPTION
**Pull request by giuseppe**

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -47,7 +47,7 @@ jobs:
         run: |
           cd $(go env GOPATH)/src/github.com/cri-o/cri-o
           make all test-binaries
-          # skip seccomp tests because they have permission denied issues in a container
-          sudo -E test/test_runner.sh $(ls test/ | grep bats | grep -v seccomp)
+          # skip seccomp tests because they have permission denied issues in a container and accept signed image as they don't use conmon
+          sudo -E test/test_runner.sh $(ls test/ | grep bats | egrep -v seccomp\|image)
```
Comment (eriksjolund) [inline .github/workflows/integration.yml:51]: Maybe replace `egrep` with `grep -E`?
See for example
* https://github.com/containers/podman/pull/18440
and https://unix.stackexchange.com/a/383454/9360

Comment (giuseppe) [reply .github/workflows/integration.yml]: good idea, fixed now

**Comment by alexlarsson** [comment]
LGTM

**Comment by haircommander** [comment]
thanks!

**Pull request merged**